### DISCUSSION TITLE
WPF开发内存泄漏，两个相机同时采集图像，期间有强制回收内存，但运行30分钟之后，内存会从200M增加到2G

### DISCUSSION THREAD

**Discussion by sheiling** [BYJRK/DotNet-Discussions#328] [WPF 开发]
内存泄漏，memory leak,

**Comment by BYJRK** [comment]
`GC.Collect()` 并不能处理非托管内存，你需要自己进行正确的释放

**Reply by sheiling** [reply]
已经去掉非托管内存代码，但运行30分钟之后，内存会从200M增长到1.2G，还有内存泄漏相关问题。大佬未来有规划出内存管理相关的教程嘛

**Reply by BYJRK** [reply]
如果你确定 C# 这边已经处理好了非托管资源，那会不会是你引用的 COM 库里面存在内存泄露的问题？
